app/src/androidTest/java/com/android/streetworkapp/ui/event/EventOverviewTest.kt

```diff
@@ -65,6 +69,25 @@ class EventOverviewTest {
     screenParams = LIST_OF_SCREENS.last()
 
     navigationActions = mock(NavigationActions::class.java)
+    // fullevent = event.copy(participants = 10, maxParticipants = 10)
```

Comment:
Comment: This commented part of code could perhaps be removed.

Reply:
done!